src/Model/Item.php

```diff
@@ -1851,13 +1908,11 @@ public static function performLike($item_id, $verb)
 			'unseen'        => 1,
 		];
 
-		$new_item_id = Item::insert($new_item);
+		$new_item_id = self::insert($new_item);
 
-		// @todo: Explain this block
-		if (! $item['visible']) {
-			q("UPDATE `item` SET `visible` = 1 WHERE `id` = %d",
-				intval($item['id'])
-			);
+		// If the parent item isn't visible then set it to visible
```

Comment:
We can guess this from the code, but why do we need to make it visible at this point?

Reply:
This code had been here before. I hadn't had a deeper look at the background.

Reply:
Fair enough.

Reply:
Possibly this is some left over functionality from the old Diaspora module that had hidden items under certain circumstances.